WordPress/src/main/java/org/wordpress/android/ui/posts/PostEditorAnalyticsSession.java

```diff
@@ -148,6 +150,7 @@ public void end() {
             }
             Map<String, Object> properties = getCommonProperties();
             properties.put(KEY_OUTCOME, mOutcome.toString().toLowerCase(Locale.ROOT));
+            properties.put(KEY_CAN_VIEW_EDITOR_ONBOARDING, canViewEditorOnboarding != null && canViewEditorOnboarding);
```

Comment:
Do you think the `canViewEditorOnboarding != null &&` part of this might throw off our analytics. This would mean that every time the `end` method is called from a stories post, we'll persist a `false` value (because that class passes `null`), whereas that same user would return a `true` value if they opened and closed Gutenberg?

Reply:
It's difficult to say whether it will throw off analytics, as I'm not sure if/why we would analyze non-Gutenberg (e.g. Story) post analytic events. However, I definitely agree setting it to `false` is less accurate. Good catch. 

I might say we wrap setting the property in a conditional again to avoid setting an inaccurate value. Do you think that would suffice? 
```suggestion
            if (canViewEditorOnboarding != null) {
                properties.put(KEY_CAN_VIEW_EDITOR_ONBOARDING, canViewEditorOnboarding);
            }
```

Reply:
> I might say we wrap setting the property in a conditional again to avoid setting an inaccurate value. Do you think that would suffice?

👍  That's probably what I would do.

Reply:
Addressed in https://github.com/wordpress-mobile/WordPress-Android/pull/14766/commits/335b4662bff4423e5fea328c4346a152315008d1.